src/test/java/org/jqassistant/plugin/spring/test/concept/WsIT.java

```diff
@@ -30,4 +45,62 @@ private void verify(Class<?> componentType, String conceptId, String expectedLab
             hasItem(typeDescriptor(componentType)));
         store.commitTransaction();
     }
+
+    @Test
+    void mockWebServiceServerVerifyMethod() throws Exception {
+        scanClasses(AssertExample.class, AssertExample.ExampleResultActions.class);
+
+        final Result<Concept> conceptResult = applyConcept("spring-ws:MockWebServiceServerVerifyMethod");
+        Assertions.assertThat(conceptResult.getStatus()).isEqualTo(SUCCESS);
```

Comment:
Please use a static import for assertThat.

Reply:
Done.